neuron/src/lib/Neuron/Zettelkasten/Query.hs

```diff
@@ -49,6 +50,12 @@ runZettelQuery zs = \case
 runGraphQuery :: ZettelGraph -> GraphQuery r -> r
 runGraphQuery g = \case
   GraphQuery_Id -> g
+  GraphQuery_BacklinksOf conn zid ->
+    case getZettel zid g of
+      Nothing -> []
```

Comment:
Instead of ignoring the error, this should perhaps raise the error like `runZettelQuery` currently does.

See https://github.com/srid/neuron/blob/4d20756dbb3d66eedc7665b4d37fb668f5d291de/neuron/src/lib/Neuron/Zettelkasten/Query.hs#L30-L34

Reply:
The graph query now uses `Either`.